server/repo/repo-core/src/main/java/org/eclipse/vorto/repository/service/IModelRepository.java

```diff
@@ -67,6 +68,15 @@
 	UploadModelResult upload(byte[] content, String fileName);
 	
 	/**
+	 * Uploads multiple model files in zip and validates it.
+	 * If all the models are valid then the models are checked in using array of {@code} ModelHandle 
+	 * {@link IModelRepository#checkinMultiple(ModelHandle)}}
+	 * @param fileName
+	 * @return result about information of the uploaded content and the upload handle. 
+	 */
+	List<UploadModelResult> uploadMultipleModels(String fileName);
```

Comment:
Why can't we re-use the existing upload(byte[] content, String fileName) ? 
I would assume that the method implementation could use 2 different handle strategies based on the fileName type ,one for zip with multiple files and one for a single model file.